open-media-match/src/OpenMediaMatch/migrations/versions/9d85c5274e1f_.py

```diff
@@ -0,0 +1,88 @@
+"""empty message
+
+Revision ID: 9d85c5274e1f
+Revises: b44643ee7ef0
+Create Date: 2023-09-29 22:42:30.984931
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "9d85c5274e1f"
+down_revision = "b44643ee7ef0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
Are we relying on database.upgrade() to create our tables instead of create_tables()?